test/bdd/steps/lib/http-api-helper.js

```diff
@@ -102,6 +102,42 @@ async function apiImport(nodeRpcUrl, importFilePath, importType) {
     });
 }
 
+/**
+ * @typedef {Object} Import
+ * @property {string} data_set_id Data-set ID.
+ * @property {string} message Message about successful import.
+ * @property {string} wallet Data provider wallet.
+ */
+
+/**
+ * Fetch /api/import
+ *
+ * @param {string} nodeRpcUrl URL in following format http://host:port
+ * @param {string} content
+ * @param {string} importType
+ * @return {Promise.<Import>}
+ */
+async function apiImportContent(nodeRpcUrl, content = '', importType = 'GS1') {
```

Comment:
@simonovic86 on the first ball, I have hard time understanding why there is a need for apiImportContent(), considering that we already have apiImport(), that could have been refactored to include default parameters values?

Reply:
It is the same route as you can see. I don't want to pass a real file in order to test if the authorization is working or not. `apiImport` helper function has hardcoded some paths which is really bad and I don't want to use it.

```importfile: fs.createReadStream(path.join(__dirname, '../../../../', importFilePath)),```